src/masternode-budget.cpp

```diff
@@ -1119,7 +1119,7 @@ CBudgetProposal::CBudgetProposal(const CBudgetProposal& other)
 
 bool CBudgetProposal::IsValid(std::string& strError)
 {
-    if(GetYeas()+GetNays() < -(mnodeman.CountEnabled()/10)){
+    if(GetYeas()+GetNays() < (mnodeman.CountEnabled()/10)){
```

Comment:
The original code is indeed incorrect, but I think this is still better:

if(GetYeas()-GetNays() < -(mnodeman.CountEnabled()/10))